src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts

```diff
@@ -112,6 +118,10 @@ export class HearingActualAddEditSummaryComponent implements OnInit, OnDestroy {
 
     if (this.hearingResult === HearingResult.CANCELLED || this.isValid()) {
       this.hearingStore.dispatch(new fromHearingStore.SubmitHearingActuals(this.id));
+      if (this.isPaperHearing) {
+        this.hearingRequestMainModel.hearingDetails.hearingChannels = [HearingChannelEnum.ONPPR.toString()];
+        this.hearingStore.dispatch(new fromHearingStore.UpdateHearingRequest(this.hearingRequestMainModel, this.hearingConditions));
```

Comment:
We actually don't need hearingConditions when we update the hearingRequest, ideally we need to use the current new fromHearingStore.SubmitHearingActuals, to call the additional service this.hearingsService.updateHearingRequest(payload), and we need to handle the errors corectly if the error is coming back either from updateHearingRequest or submitHearingActuals.